src/platform/site-wide/sass/_b-brand-consolidation-colors.scss

```diff
@@ -0,0 +1,2 @@
+$red-70v: #8b1303;
```

Comment:
I don't know what to do with these at the moment.

Reply:
Cross-linking to work in another PR that add color variables - https://github.com/department-of-veterans-affairs/vets-website/pull/8600/files#diff-d2a132356cbec2434ff0d2a6301d6ed6R15